src/ddmd/parse.d

```diff
@@ -4767,6 +4767,13 @@ final class Parser(AST) : Lexer
             else if (!f.frequire && !f.fensure) // allow these even with no body
             {
                 error("semicolon expected following function declaration");
```

Comment:
How about moving this to the `else` - i.e. as default message?

Reply:
fair enough